packages/uswds-core/src/js/utils/validate-input.js

```diff
@@ -14,20 +14,52 @@ module.exports = function validate(el) {
     throw new Error(`No validation element found with id: "${id}"`);
   }
 
+  let statusSummary = "";
   Object.entries(el.dataset).forEach(([key, value]) => {
     if (key.startsWith("validate")) {
       const validatorName = key.substr("validate".length).toLowerCase();
       const validatorPattern = new RegExp(value);
       const validatorSelector = `[data-validator="${validatorName}"]`;
       const validatorCheckbox = checkList.querySelector(validatorSelector);
+      const validatorParent = el.parentNode;
+      const statusSummaryContainer = validatorParent.querySelector(`[data-validation-status]`);
+
+      const checked = validatorPattern.test(el.value);
+      validatorCheckbox.classList.toggle(CHECKED_CLASS, checked);
 
       if (!validatorCheckbox) {
         throw new Error(`No validator checkbox found for: "${validatorName}"`);
       }
 
-      const checked = validatorPattern.test(el.value);
-      validatorCheckbox.classList.toggle(CHECKED_CLASS, checked);
-      validatorCheckbox.setAttribute(CHECKED, checked);
+      // Create status reports for checklist items
+      let statusComplete = "status complete";
+      let statusIncomplete = "status incomplete";
+      let checkboxContent = `${validatorCheckbox.textContent} `;
+
+      if (el.hasAttribute("data-validation-complete")) {
+        statusComplete = el.getAttribute("data-validation-complete");
+      }
+
+      if (el.hasAttribute("data-validation-incomplete")) {
+        statusIncomplete = el.getAttribute("data-validation-incomplete");
+      }
+
+      if(validatorCheckbox.classList.contains(CHECKED_CLASS)){
+        checkboxContent += statusComplete;
+      } else{
+        checkboxContent += statusIncomplete;
+      };
+
+      // move status updates to aria-label on checklist item
+      validatorCheckbox.setAttribute("aria-label", checkboxContent);
+
+      // Create a summary of status for all checklist items
+      statusSummary += `${checkboxContent}. `;
+
+      // Add summary to screen reader summary container, after a delay
+      ((debounce(() => {
+        statusSummaryContainer.textContent = statusSummary;
+      }, 1000)))();
```

Comment:
Optional because we have the comment, but we can increase readability by extracting it to it's own `updateSRSummary()` and just calling it here.

Reply:
I like that it could make it more consistent with the new character count setup. Updated in [bfbfd52](https://github.com/uswds/uswds/pull/4914/commits/bfbfd52063589b239ac2bc3c215dd86c09c7bc5f)

(If I misunderstood your request, feel free to fix while I am out!)